webpack.config.js

```diff
@@ -1,174 +1,68 @@
-        try {
-            fs.readdirSync(dir, {withFileTypes: true}).forEach(function(item) {
-                const file = item.name;
-                const parts = file.split('.');
-                if (parts[1] !== 'js') return;
-                var file_path = dir + '/' + file;
-                entry_points[parts[0]] = file_path;
-            });
-        } catch (e) {
-            console.log(e);
-        }
-    }
-
-    each_file(dir);
-
-    return entry_points;
-}
-
-function generateHtmlPlugins(templateDir) {
-    const templateFiles = fs.readdirSync(path.resolve(__dirname, templateDir), {withFileTypes: true});
-
-    return templateFiles.reduce((result, item) => {
-        const parts = item.name.split('.');
-        const name = parts[0];
-        const extension = parts[1];
-
-        if (extension === 'html') result.push(
-            new HtmlWebpackPlugin({
-                filename: path.resolve(__dirname, `./theme/html/${name}.html`),
-                template: path.resolve(__dirname, `${templateDir}/${name}.${extension}`),
-                chunks: ['style', 'vendors', 'global', name],
-                chunksSortMode: 'manual',
-                excludeAssets: 'style.js'
-            })
-        );
-       
-        return result;
-    }, []);
-}
-
-function plugins() {
-    let plugins = [...generateHtmlPlugins('./src/pages/')];
-
-    if (isDev) {
-        plugins.push(new webpack.HotModuleReplacementPlugin());
-    } else {
-        plugins = [
-            ...plugins,
-            new HtmlBeautifyPlugin({
-                config: {
-                    html: {
-                        end_with_newline: true,
-                        indent_size: 4,
-                        indent_with_tabs: true,
-                        indent_inner_html: true,
-                        preserve_newlines: true,
-                        unformatted: ['strong', 'i', 'b'],
-                        inline: []
-                    }
-                }
-            }),
-            new HtmlWebpackSkipAssetsPlugin({
-                excludeAssets: ['style.js']
-            }),
-            new RemoveEmptyScriptsPlugin(),
-            new MiniCssExtractPlugin({
-                filename: ({chunk}) => chunk.name === 'style' ? '[name].css' : 'css/[name].css'
-            })
-        ]
-    }
-    
-    return plugins;
-}
-
-function processNestedHtml(content, loaderContext) {
-    const INCLUDE_PATTERN = /\<include src=\"(.+)\"\/?\>(?:\<\/include\>)?/gi;
-    
-    return !INCLUDE_PATTERN.test(content) 
-    ? content 
-    : content.replace(INCLUDE_PATTERN, (m, src) => {
-        const pathToPartial = path.resolve(loaderContext.context, src);
-        loaderContext.addDependency(pathToPartial);
-        return processNestedHtml(fs.readFileSync(pathToPartial, 'utf8'), {...loaderContext, context: path.dirname(pathToPartial)});
-    });
-}
-
-function styleLoaders() {
-    if (isDev) {
-        return [
-            "style-loader",
```

Comment:
The `HtmlBundlerPlugin` can inline extracted CSS into HTML,
no style-loader is required.